recipe/meta.yaml

```diff
@@ -10,18 +11,24 @@ source:
   sha256: a84f4bb7c5da1fc396ac3ebf493fa470a2cf208924b4bd9c6321d65d4906df9e
 
 build:
+  number: 1
   noarch: python
-  number: 0
```

Comment:
Bump the `build/number` as the Python minimum version was updated from 3.5 to 3.6 to reflect [the upstream requirement]( https://github.com/gpuopenanalytics/pynvml/blob/11.5.3/setup.py#L16 ). So new packages are needed